workers/varcode_annotator.py

```diff
@@ -0,0 +1,133 @@
+"""Annotate genotype data with the effects on gene- and protein-level
+
+Given genotype data set that exists in the DB, finds the transcript- and
+protein-level effects of variants and adds them as annotations.
+
+The worker utilizes a combination of PyEnsembl and Varcode methods
+to do the annotations and save them back into the database.
+
+Steps:
+
+    1. Get contig and position data from the genotypes table.
+    2. Look up each (contig, position, ref, alt) from Varcode
+    3. Create a CSV file, and write the annotations to it.
+    4. Create a temporary DB table, and write the CSV file to it.
+    5. Update the genotypes table by joining with the temporary table.
+    6. Update the list of extant columns.
+"""
+from contextlib import contextmanager
+import csv
+import re
+import sqlalchemy
+from sqlalchemy import select, Table, Column
+from sqlalchemy.types import Text, Integer
+
+import config
+
+from common.helpers import tables
+
+from workers.shared import (worker, DATABASE_URI, TEMPORARY_DIR,
+                            initialize_database, temp_csv,
+                            update_extant_columns, register_running_task)
+
+if not config.TRAVIS:
+    from pyensembl import EnsemblRelease
+    from varcode import Variant
+
+@worker.task(bind=True)
+def annotate(self, vcf_id):
+    if vcf_id == False:
+        return  # An error must have occurred earlier.
+    register_running_task(self, vcf_id)
+
+    EnsemblRelease(config.ENSEMBL_RELEASE).install()  # Only runs the first time for this release.
+
+    engine = sqlalchemy.create_engine(DATABASE_URI)
+    with tables(engine, 'genotypes') as (con, genotypes):
+        metadata = sqlalchemy.MetaData(bind=con)
+        metadata.reflect()
+        gene_names = get_varcode_annotations(genotypes, vcf_id, config.ENSEMBL_RELEASE)
```

Comment:
Maybe `gene_names` should be `annotations` or something? In general, you may consider dropping the `varcode_` prefix you have in some places. I don't think it's necessary (though I can see the argument & use for having it in the module name, as this module just annotates the loci with all the annotations Varcode generates).